java/Oleksii-Riabov/src/com/zoolatech/lecture2/tasks/_2/Controller.java

```diff
@@ -0,0 +1,172 @@
+package com.zoolatech.lecture2.tasks._2;
+
+import java.util.Scanner;
+
+public class Controller {
+    Robot robot;
+
+    public void create() {
+        System.out.println("To create Robot with default values enter \"1\"" +
+                "\nTo create Robot with values initialization enter \"2\"");
+
+        while (true) {
+            Scanner scanner = new Scanner(System.in);
+            int inputValue;
+            if (scanner.hasNextInt()) {
+                inputValue = scanner.nextInt();
+                if (inputValue == 1) {
+                    robot = new Robot();
+                    break;
+                } else if (inputValue == 2) {
+                    System.out.println("Enter Room's width as integer value.");
+                    int roomWidth = getValue();
+                    roomWidth--;
+
+                    System.out.println("Enter Room's height as integer value.");
+                    int roomHeight = getValue();
+                    roomHeight--;
+
+                    System.out.println("Enter Robot's default direction as String value (south, west, north or east).");
+                    String direction;
+                    outer:
+                    while (true) {
```

Comment:
try to rewrite the loop without while(true) and label outer

Reply:
Already replace "while(true)" everywhere.